team/bundles/org.eclipse.compare/compare/org/eclipse/compare/internal/merge/DocumentMerger.java

```diff
@@ -403,7 +402,6 @@ public void doDiff() throws CoreException {
 		final Object[] result= new Object[1];
 		final DocLineComparator sa= sancestor, sl= sleft, sr= sright;
 		IRunnableWithProgress runnable= monitor -> {
-			monitor.beginTask(CompareMessages.DocumentMerger_0, maxWork(sa, sl, sr));
```

Comment:
Can we instead have this:
`monitor.setTaskName(CompareMessages.DocumentMerger_0);`

Otherwise the progress dialog shows just "Operation in progress..."  instead of "Computing Differences...".

Same below.

Reply:
With breakpoint after SubMonitor.convert on RangeComparatorLCS.findDifferences()
It's
![image](https://github.com/eclipse-platform/eclipse.platform/assets/51790620/f9360f5e-8b21-491b-b3ed-722768d4d6d3)
Is it really worth to show another text for the first few nanoseconds?

Reply:
> With breakpoint after SubMonitor.convert on RangeComparatorLCS.findDifferences()

We could save our time if the commit would mention that the task name is reported later on. I had breakpoint in the changed lambda...